<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tasky</class>
 <widget class="QMainWindow" name="tasky">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>tasky</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">background-color: rgb(37, 37, 37);
color: rgb(246, 245, 244);</string>
   </property>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>391</x>
      <y>9</y>
      <width>431</width>
      <height>261</height>
     </rect>
    </property>
    <property name="title">
     <string>Lista de Tareas</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QTableWidget" name="tblTareas"/>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>376</width>
      <height>259</height>
     </rect>
    </property>
    <property name="title">
     <string>Agregar Tarea</string>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Tarea:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="txtTarea">
       <property name="placeholderText">
        <string>Ingrese una tarea</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Asignatura:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="cmbAsignatura">
       <item>
        <property name="text">
         <string>Cálculo Integral</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Ética</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Estructura de Datos</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Electrotecnia</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Programación Orientada a Objetos</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Fecha:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QDateEdit" name="dFecha">
       <property name="dateTime">
        <datetime>
         <hour>0</hour>
         <minute>0</minute>
         <second>0</second>
         <year>2023</year>
         <month>9</month>
         <day>14</day>
        </datetime>
       </property>
       <property name="minimumDateTime">
        <datetime>
         <hour>0</hour>
         <minute>0</minute>
         <second>0</second>
         <year>2023</year>
         <month>9</month>
         <day>14</day>
        </datetime>
       </property>
       <property name="displayFormat">
        <string>d/M/yyyy</string>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Hora:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QTimeEdit" name="dHora">
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QWidget" name="widget" native="true">
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <layout class="QFormLayout" name="formLayout_2">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>170</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="btnAgregar">
          <property name="text">
           <string>Agregar</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="7" column="1">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="backgroundButton">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>290</y>
      <width>268</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Dark/Light Theme</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
